### PR TITLE
refactor(style-MultiSelectEntries): adapt multi select entries to the dark mode colors

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
adapt multi select entries to the dark mode colors

### CODE REVIEW

**Comment by bdauria** [comment]
@Noha-ElAbrouki can you add a few screenshots (for both light and dark) so that we all see what it looks like?

File: packages/centreon-ui/src/MultiSelectEntries/index.tsx [hunk 1]
```diff
@@ -35,7 +36,17 @@ const useStyles = makeStyles((theme) => ({
     visibility: 'hidden',
   },
   hovered: {
-    backgroundColor: theme.palette.grey[400],
+    backgroundColor: theme.palette.action.hover,
+  },
+  iconColer: {
```
Comment (bdauria) [inline packages/centreon-ui/src/MultiSelectEntries/index.tsx:41]:
```suggestion
  icon: {
```

File: packages/centreon-ui/src/MultiSelectEntries/index.tsx [hunk 2]
```diff
@@ -111,7 +132,10 @@ const MultiSelectEntries = ({
         </Box>
         <Box>
           <IconCreate
-            className={clsx({ [classes.hidden]: !isHovered && !highlight })}
+            className={clsx(
+              { [classes.hidden]: !isHovered && !highlight },
+              classes.iconColer,
```
Comment (bdauria) [inline packages/centreon-ui/src/MultiSelectEntries/index.tsx:137]:
```suggestion
              classes.icon,
```

File: packages/centreon-ui/src/MultiSelectEntries/index.tsx [hunk 3]
```diff
@@ -35,7 +36,17 @@ const useStyles = makeStyles((theme) => ({
...
+    backgroundColor: theme.palette.action.hover,
+  },
+  iconColer: {
+    color: theme.palette.action.active,
+  },
+  labelChip: {
+    color: theme.palette.text.disabled,
+  },
+
+  textColor: {
```
Comment (bdauria) [inline packages/centreon-ui/src/MultiSelectEntries/index.tsx:48]:
```suggestion
  text: {
```

File: packages/centreon-ui/src/MultiSelectEntries/index.tsx [hunk 4]
```diff
@@ -59,9 +74,15 @@ const EmptyEntry = ({ label }): JSX.Element => {
...
 };
 
-const Caption = ({ children }): JSX.Element => (
-  <Typography variant="caption">{children}</Typography>
-);
+const Caption = ({ children }): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Typography className={classes.textColor} variant="caption">
```
Comment (bdauria) [inline packages/centreon-ui/src/MultiSelectEntries/index.tsx:81]:
```suggestion
    <Typography className={classes.text} variant="caption">
```

**Comment by Noha-ElAbrouki** [comment]
> @Noha-ElAbrouki can you add a few screenshots (for both light and dark) so that we all see what it looks like?
EmptyLight

emptyDark
onElementsHighliLight
onElementHighDark
sixElementLight
sixElementsDark

**Review by bdauria — APPROVED** [review]

File: packages/centreon-ui/src/MultiSelectEntries/index.tsx [hunk 5]
```diff
@@ -35,7 +36,17 @@ const useStyles = makeStyles((theme) => ({
...
-    backgroundColor: theme.palette.grey[400],
+    backgroundColor: theme.palette.action.hover,
+  },
+  icon: {
+    color: theme.palette.action.active,
+  },
+  labelChip: {
+    color: theme.palette.text.disabled,
+  },
+
```
Comment (Thebarda) [inline packages/centreon-ui/src/MultiSelectEntries/index.tsx:47]:
```suggestion
```

**Pull request merged**